dummy.js

```diff
@@ -0,0 +1,9 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+$('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://dev.k8s.dev.codacy.org/gh/DevK8scodacytestuser/CheckPrCommentsPublicProjectStaging/pullRequest?prid=2442)
```suggestion
$(".jsSelectedMainEmail").text(postVariable.email);
```